internal/api/api.go

```diff
@@ -67,6 +68,7 @@ func New(
 	api.RegisterHandlerOnPrefix("/debug", api.healthHandler())
 	api.router.Handle("/", http.RedirectHandler(login.HandlerPrefix, http.StatusFound))
 
+	reflection.Register(api.grpcServer)
```

Comment:
Do we have to ignore some paths in the `server.CreateServer` function?

Reply:
I've added the reflection path to the Probes list. Please check if this is ok.